<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tkeDialogDateInterval</class>
 <widget class="QDialog" name="tkeDialogDateInterval">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Задання інтервалу</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_begin_date">
       <property name="text">
        <string>Початкова дата</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditPochDate">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximumDate">
        <date>
         <year>2100</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>2007</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>MM.yyyy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_end_date">
       <property name="text">
        <string>Кінцева дата</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditKincDate">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximumDate">
        <date>
         <year>2100</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>2007</year>
         <month>1</month>
         <day>14</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>MM.yyyy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_accept">
       <property name="text">
        <string>Прийняти</string>
       </property>
       <property name="icon">
        <iconset resource="../tkeAbonent/tke/tke_rc">
         <normaloff>:/pictures/pictures/button_ok.png</normaloff>:/pictures/pictures/button_ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_reject">
       <property name="text">
        <string>Відхилити</string>
       </property>
       <property name="icon">
        <iconset resource="../tkeAbonent/tke/tke_rc">
         <normaloff>:/pictures/pictures/button_cancel.png</normaloff>:/pictures/pictures/button_cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../tkeAbonent/tke/tke_rc"/>
  <include location="../tkeAbonent/tke/tke_rc"/>
 </resources>
 <connections/>
</ui>
